src/main/java/codeu/model/data/Message.java

```diff
@@ -66,13 +69,28 @@ public String getContent() {
     return content;
   }
 
+ /** Returns mentioned user. */
+  public String getMentionedUser(){
+    return mentionedUser;
```

Comment:
I think a message should be allowed to have multiple mentions. This is pretty frequent in messaging apps